src/components/LinterMessage/index.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import unescape from 'lodash.unescape';
+import { Alert } from 'react-bootstrap';
+
+import styles from './styles.module.scss';
+import { LinterMessage as LinterMessageType } from '../../linter';
+
+const getAlertVariant = (type: LinterMessageType['type']) => {
+  switch (type) {
+    case 'error':
+      return 'danger';
+    case 'warning':
+      return 'warning';
+    default:
+      return 'secondary';
+  }
+};
+
+const unescapeHtmlEntities = (text: string) => {
+  // This unescapes HTML entities (like &lt;) so that React will
+  // escape them again. Without this, React will double escape the
+  // HTML entities
+  return unescape(text);
+};
+
+const renderDescription = (description: LinterMessageType['description']) => {
+  const urlPattern = /^https?:\/\//;
+
+  return description.reduce((allLines: React.ReactNode[], line, lineIndex) => {
+    if (lineIndex > 0) {
+      allLines.push(<br key={line} />);
+    }
+
+    allLines.push(
+      // Intercept and replace URLs with JSX links.
+      line
+        .trim()
+        .split(/(\s+)/)
+        .reduce((allParts: React.ReactNode[], part) => {
+          if (urlPattern.test(part)) {
+            allParts.push(<Alert.Link href={part}>{part}</Alert.Link>);
```

Comment:
Don't we need a `key` here too? 🤔

Reply:
Will these links be internal or external? I am thinking about react-router and target=_blank.

Reply:
They will be external, yeah

Reply:
I will file an issue to make the links external because I don't see an obvious way to do it